File: go/01-Saying-Hello/hello.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)

	fmt.Print("What is your name? ")
	name, _ := reader.ReadString('\n')
	name = strings.Replace(name, "\n", "", -1)

	output := "Hello, " + name + ", nice to meet you!"

	fmt.Println(output)
}

File: python/01-saying-hello/hello.py
#!/usr/bin/env python3
"""
Ask the user for their name and print a greeting.
"""


def main():
    """
    Ask the user for their name and print a greeting.
    """
    name = input("What is your name? ")
    output = "Hello, " + name + ", nice to meet you."
    print(output)


if __name__ == '__main__':
    main()
